src/index.ts

```diff
@@ -21,6 +22,7 @@ import {
   readCoverageFile,
   removeFile,
 } from './helpers/files'
+import { arch } from 'os'
```

Comment:
```suggestion
```

This appears to be unused.

Reply:
I don't even know what it's doing there 🤷‍♀️

Reply:
VS Code autocomplete probably brought it in.